### DISCUSSION TITLE
Kotlin Language

### DISCUSSION THREAD

**Discussion by OurCodeBase** [Rosemoe/sora-editor#229] [General]
Hello @Rosemoe
I want to use kotlin as a language instead of `editor.setEditorLanguage(new JavaLanguage());`
How can i do that ?

**Comment by Rosemoe** [comment]
Use textmate with Kotlin syntax files

**Reply by OurCodeBase** [reply]
### Thanks to `@Rosemoe` for reply.
Can you share any example of it or explain more deeply ?? I didn't understand.
